pkg/types/ccipocr3/interfaces.go

```diff
@@ -20,11 +20,9 @@ type MessageHasher interface {
 	Hash(context.Context, Message) (Bytes32, error)
 }
 
-type ExtraDataCodec interface {
-	// DecodeExtraArgs reformat bytes into a chain agnostic map[string]any representation for extra args
-	DecodeExtraArgs(extraArgs Bytes, sourceChainSelector ChainSelector) (map[string]any, error)
-	// DecodeTokenAmountDestExecData reformat bytes to chain-agnostic map[string]any for tokenAmount DestExecData field
-	DecodeTokenAmountDestExecData(destExecData Bytes, sourceChainSelector ChainSelector) (map[string]any, error)
+type AddressCodec interface {
+	AddressBytesToString([]byte) (string, error)
```

Comment:
We would need to also pass in the ChainSelector param, I think.
All the ccip code which calls this interface may not know or have this codec instance tied to a chain in my opinion.